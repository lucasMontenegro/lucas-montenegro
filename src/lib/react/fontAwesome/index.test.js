describe(`./index.js`, () => {
  it(`should use the right versions of its dependencies`, () => {
    expect(jestUtils.getDependencies([
      `@fortawesome/fontawesome-svg-core`,
      `@fortawesome/free-brands-svg-icons`,
      `@fortawesome/free-solid-svg-icons`,
      `@fortawesome/free-regular-svg-icons`,
      `@fortawesome/react-fontawesome`,
    ])).toMatchSnapshot()
  })
  it(`should run`, () => {
    require("./index.js")
  })
})