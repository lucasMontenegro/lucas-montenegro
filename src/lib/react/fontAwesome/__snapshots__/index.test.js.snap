// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`./index.js should use the right versions of its dependencies 1`] = `
Object {
  "@fortawesome/fontawesome-svg-core": "^1.2.25",
  "@fortawesome/free-brands-svg-icons": "^5.11.2",
  "@fortawesome/free-regular-svg-icons": "^5.11.2",
  "@fortawesome/free-solid-svg-icons": "^5.11.2",
  "@fortawesome/react-fontawesome": "^0.1.5",
}
`;
